tests/perf/conftest.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import pytest
+import os
+import subprocess
+import yaml
+from pathlib import Path
+from typing import List
+from datetime import datetime
+
+from otx.api.entities.model_template import ModelTemplate, ModelCategory
+from .benchmark import OTXBenchmark
+
+
+def pytest_addoption(parser):
+    """Add custom options for perf tests."""
+    parser.addoption(
+        "--model-type",
+        action="store",
+        default="all",
+        choices=("default", "all"),
+        help="Choose default|all. Defaults to all.",
+    )
+    parser.addoption(
+        "--data-size",
+        action="store",
+        default="all",
+        choices=("small", "medium", "large", "all"),
+        help="Choose small|medium|large|all. Defaults to all.",
+    )
+    parser.addoption(
+        "--num-repeat",
+        action="store",
+        default=0,
+        help="Overrides default per-data-size number of repeat setting. "
+        "Random seeds are set to 0 ~ num_repeat-1 for the trials. "
+        "Defaults to 0 (small=3, medium=3, large=1).",
+    )
+    parser.addoption(
+        "--num-epoch",
+        action="store",
+        default=0,
+        help="Overrides default per-model number of epoch setting. "
+        "Defaults to 0 (per-model epoch & early-stopping).",
+    )
+    parser.addoption(
+        "--eval-upto",
+        action="store",
+        default="all",
```

Comment:
```suggestion
        default="train",
```
Maybe it's a typo?

Reply:
Fixed in https://github.com/openvinotoolkit/training_extensions/pull/2742/commits/91bfec13541e70258511473e29b8b56dfed4c3d1